chain/index/indexer.go

```diff
@@ -120,15 +136,18 @@ func NewSqliteIndexer(path string, cs ChainStore, gcRetentionEpochs int64, recon
 func (si *SqliteIndexer) Start() error {
 	si.wg.Add(1)
 	go si.gcLoop()
+
+	si.started = true
+
 	return nil
```

Comment:
just noticed the lack of an error to return here from seeing `_ = si.Start()` in the tests, how about we just remove this, unless we're trying to conform to some interface?

Reply:
Done.